src/start.js

```diff
@@ -5,53 +5,67 @@ const startInstances = require('./instances/startInstances');
 const listInstancesToStart = require('./instances/listInstancesToStart');
 const untagInstances = require('./instances/untagInstances');
 
-function startAllInstances() {
+function startAllInstances(dryRun) {
   return listInstancesToStart()
     .then((startableInstances) => {
-      console.log(`Found the following ${startableInstances.length} instances to start up...`);
+      if (dryRun) {
+        console.log('Dry run is enabled, will not start or untag any instances.');
+        return [];
+      }
+
       if (startableInstances.length === 0) {
-        console.log('None! Moving on.');
-        return 'No instances to turn on';
+        console.log('No instances found to start, moving on...');
+        return [];
       }
 
+      console.log(`Found the following ${startableInstances.length} instances to start up...`);
       startableInstances.forEach((instance) => {
         console.log(instance);
       });
-      return startInstances(startableInstances);
-    })
-    .then((startedInstances) => {
-      console.log('Finished starting instances. Moving on to untag them.');
-      return untagInstances(startedInstances);
+
+      return startInstances(startableInstances).then((startedInstanceIds) => {
```

Comment:
If instances were previously started & the app stopped for what-ever reason this removes the ability to have Hammertime remove the tags.